app/src/main/java/com/github/libretube/helpers/ImportHelper.kt

```diff
@@ -45,26 +48,43 @@ object ImportHelper {
      * Get a list of channel IDs from a file [Uri]
      */
     @OptIn(ExperimentalSerializationApi::class)
-    private fun getChannelsFromUri(activity: Activity, uri: Uri): List<String> {
+    private fun getChannelsFromUri(activity: Activity, uri: Uri, importFormat: ImportFormat): List<String> {
         return when (val fileType = activity.contentResolver.getType(uri)) {
             "application/json", "application/*", "application/octet-stream" -> {
```

Comment:
I think we should just drop that lines entirely (the file type check) and just act depending on the selected import format by the user only.